### PR TITLE
Confirm service solicitor event

### PR DESCRIPTION
**Pull request by mounikahmcts**
https://tools.hmcts.net/jira/browse/DIV-4963

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -4737,5 +4737,33 @@
     "CaseEventID": "solicitorSOTDN",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solConfirmPersonalService",
+    "UserRole": "[PETSOLICITOR]",
```
Comment (alexwakeman) [inline definitions/divorce/json/AuthorisationCaseEvent.json]: Hi Mounika - we have withdrawn the [PETSOLICITOR] role until after we can fully test this area, given we're not pursuing the current approach of linking. As it is not in the next release (probably) then we should hold back merging this stuff...

Comment (mounikahmcts) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: Hi @alexwakeman  will make a note of it.  Do we need to use normal caseworker solicitor role in this case? Will check with @gavinclark  before merging this.

Comment (alexwakeman) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: Would use caseworker-divorce-solicitor as we would have done previously

Comment (mounikahmcts) [reply definitions/divorce/json/AuthorisationCaseEvent.json]: > Would use caseworker-divorce-solicitor as we would have done previously

Moved the permissions to caseworker-divorce-solicitor role

**Pull request merged**